src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java

```diff
@@ -7,34 +7,22 @@
  */
 package com.signicat.hystrix.servlet;
 
-import com.netflix.hystrix.Hystrix;
```

Comment:
Clean the imports that need to be cleaned. The rest: Don't introduce unneccessary noise.